en/docs/InstallAndSetup/UpgradingWSO2APIManager/upgrading-from-191-to-300.md

```diff
@@ -1,34 +1,39 @@
-# Upgrading API Manager from 1.9.1 to 3.0.0
+# Upgrading API Manager from 1.8.0/1.9.0/1.9.1 to 3.0.0
 
-The following information describes how to upgrade your API Manager server **from APIM 1.8.0/1.9.0/1.9.1 to 3.0.0** .
+The following information describes how to upgrade your API Manager server **from APIM 1.8.0/1.9.0/1.9.1 to 3.0.0**.
 
 !!! note
-To upgrade **from a version older than 1.8.0** , follow the instructions in the document that was released immediately after your current release and upgrade incrementally.
+    Before you follow this section, see [Upgrading Process](../UpgradingWSO2APIManager/upgrading-process.md) for more information.
 
-!!! tip
-    Before you begin
-
-    - This release is a WUM-only release. This means that there are no manual patches. Any further fixes or latest updates for this release can be updated through the WSO2 Update Manager (WUM).
+!!! attention "Before you Begin"
+    1.  This release is a WUM-only release. This means that there are no manual patches. Any further fixes or latest updates for this release can be updated through the WSO2 Update Manager (WUM).
 
-        -   **If you are upgrading to this version, in order to use this version in your production environment** , use the WSO2 Update Manager and get the latest available updates for WSO2 API Manager 3.0.0. For more information on how to do this, see [Updating WSO2 Products](https://docs.wso2.com/display/ADMIN44x/Getting+Started+with+WUM).
+        -   **If you are upgrading to this version, in order to use this version in your production environment** , use the WSO2 Update Manager and get the latest available updates for WSO2 API Manager 3.0.0. For more information on how to do this, see [Updating WSO2 Products](https://docs.wso2.com/display/updates/Using+WSO2+Update+Manager).
 
-    - Before starting the upgrade, run the [token and session cleanup scripts](../../Administer/ProductAdministration/removing-unused-tokens-from-the-database.md) in the databases of the environment, if you are not doing regular cleanups.
+    2.  Before starting the upgrade, run the [token and session cleanup scripts](../../Administer/ProductAdministration/removing-unused-tokens-from-the-database.md) in the databases of the environment, if you are not doing regular cleanups.
```

Comment:
Did we verify the scripts against older API Manager versions (1.9.1 / 1/10.0)? There might be changes in identity tables in older versions.. Since we are anyway having that note in 2.0.0, shall we remove this from here?
If the scripts are working in lower versions, we can keep as it is.

Reply:
We have a note in the pages in the 2.6 doc space as https://docs.wso2.com/display/AM260/Upgrading+from+the+Previous+Release#191

Since it's a recommendation to run the cleanup scripts before a migration, shall we keep the note as it is?

Reply:
As per the offline discussion, we will keep the note for token cleanup.